tsconfig.build.json

```
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.common",
+  "exclude": ["**/__tests__/**", "*.unit.spec.ts"]
```

```suggestion
  "exclude": ["**/__tests__/**", "**/*.spec.ts", "**/*.test.ts"]
```
we have integrations as well